OWNERS

```diff
@@ -1,21 +1,13 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-  - durandom
-  - tumido
-  - HumairAK
+  - coghlanRH
+  - michaelclifford
 
 reviewers:
```

Comment:
@MichaelClifford Please add users akrawczy, christusa

Reply:
Those new members were requested in PR https://github.com/operate-first/common/pull/50